.github/workflows/main.yml

```diff
@@ -15,7 +15,7 @@ jobs:
           - { jdk: 17, idea: 2023.2 }
           - { jdk: 17, idea: 2023.3 }
           - { jdk: 17, idea: 2024.1 }
-          - { jdk: 17, idea: LATEST-EAP-SNAPSHOT }
+          - { jdk: 21, idea: 242-EAP-SNAPSHOT }
```

Comment:
Does it make a difference whether we use `LATEST-EAP-SNAPSHOT` vs `242-EAP-SNAPSHOT`?

Reply:
At some point `LATEST-EAP-SNAPSHOT` was an alias of `241-EAP-SNAPSHOT`, not `242-EAP-SNAPSHOT`.
That's no longer the case